fluidefi/fluidefi-caspernet-aggregator-tools
# [WIP] Event Mapper Service

**Summary of change:**
- Added a new micro-service, deploys finder, which accepts block number as an argument and returns the deploy. 

**List of tasks:** 
- Deploy Finder

**Steps for testing**
- Make sure your local git is up to date
```bash
git pull
```

- Checkout the branch `feature/deploy-finder`
```bash
git checkout feature/deploy-finder
```

- Run the test across the test-net
```bash
npm run test
```

- Confirm deploy-finder test has passed.
